server.js

```diff
@@ -1,11 +0,0 @@
-const
-  express = require('express'),
```

Comment:
Since we're not going to be using SSR at the moment, can you revert this change? The SSR scripts can be kept, we just need this file for Heroku

Reply:
Sure.